simplescreenrecorder.spec

```diff
@@ -0,0 +1,89 @@
+%global shortname ssr
+
+%undefine __cmake_in_source_build
+
+Name:           simplescreenrecorder
+Version:        0.4.5
+Release:        1%{?dist}
+Summary:        Simple Screen Recorder is a screen recorder for Linux
+
+License:        GPLv3
+URL:            https://www.maartenbaert.be/simplescreenrecorder/
+Source0:        https://github.com/SergeyMy/ssr/archive/%{version}/%{name}-%{version}.tar.gz
```

Comment:
Is this supposed to point to your fork or the main project's repository?

Reply:
> Is this supposed to point to your fork or the main project's repository?

main project's repository
only a label other than master is needed!
:)

Reply:
What command we need to use to fetch this « tag » revision of code ?

Reply:
:)
git clone https://github.com/SergeyMy/ssr.git -b 0.4.5
or
wget https://github.com/SergeyMy/ssr/archive/0.4.5/simplescreenrecorder-0.4.5.tar.gz